build.sbt

```diff
@@ -9,7 +9,7 @@ import scala.util.Try
 
 ThisBuild / version := {
   // update .travis.yml too for dog fooding
-  val v = "1.4.2-SNAPSHOT"
+  val v = "1.4.3-SNAPSHOT"
```

Comment:
We need to bump `.travis.yml` as I wrote in the comment.

Reply:
Strangely that doesn't fix it: https://travis-ci.com/github/sbt/sbt/jobs/425031122

Reply:
```bash
rm -r $(find $HOME/.sbt/boot -name "*-SNAPSHOT") || true
rm: missing operand
Try 'rm --help' for more information.
```

Reply:
Actually not sure if that would cause this.. I'm going to restart the job.